pulseapi/entries/views.py

```diff
@@ -47,6 +47,13 @@ def post_validate(request):
 
     return True
 
+class EntriesPagination(PageNumberPagination):
+    """
+    Add support for pagination and custom page size
+    """
+    page_size = 48
```

Comment:
We might want to file an issue to figure out what a manageable value is based on how many entries the client displays at a time?